schema/defs.json

```diff
@@ -141,6 +147,12 @@
                 },
                 "type": {
                     "type": "string"
+                },
+                "uidmappings": {
+                    "$ref": "#/definitions/ArrayOfIDMappings"
+                },
+                "gidmappings": {
+                    "$ref": "#/definitions/ArrayOfIDMappings"
```

Comment:
could we just use here:

```json
                "uidMappings": {
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/IDMapping"
                    }
                },
                "gidMappings": {
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/IDMapping"
                    }
                }
```

so we do not need to define the new ArrayOfIDMappings type?

Alternatively let's change the names to `uidMappings` and `gidMappings`, and also update `config-linux.json` to use the new type as well